src/Microsoft.PowerShell.ConsoleHost/host/msh/CommandLineParameterParser.cs

```diff
@@ -190,6 +190,7 @@ internal class CommandLineParameterParser
             "file",
             "executionpolicy",
             "command",
+            "settingsFile",
```

Comment:
Maybe change the name to 'settingfile'? as none of the existing flag names here use an uppercase letter.

Reply:
@daxian-dbw: I assume you are referring to the camel casing, right or are you also suggesting the singular form.  I agree with removing the camel casing but I don't think using the singular makes grammatical sense for this option.

Reply:
I was referring to the camel casing.

Reply:
fixed